GoogleTestAdapter/Tests.Common/ResultChecker/TrxResultChecker.cs

```diff
@@ -61,7 +61,8 @@ public void RunTestsAndCheckOutput(string typeName, string arguments, string tes
                 Assert.Inconclusive($"First time this test runs, created golden file - check for correctness! File: {goldenFile}");
             }
             CheckIfFileIsParsable(goldenFile);
-            if (!CompareXmlFiles(goldenFile, transformedResultFile, out string diffFile))
+            string diffFile;
```

Comment:
@jgefele what was the reason for removing C# 7.0 features? I'm not sure if we use any on our side so we would need to validate if there is a need to remove them altogether.